apps/desktop-wallet/src/api/transactions.ts

```diff
@@ -1,13 +1,12 @@
 import { keyring } from '@alephium/keyring'
-import { AddressHash, throttledClient } from '@alephium/shared'
+import { Address, AddressHash, throttledClient } from '@alephium/shared'
 
 import { LedgerAlephium } from '@/features/ledger/utils'
-import { Address } from '@/types/addresses'
 import { CsvExportQueryParams } from '@/types/transactions'
 
-export const buildSweepTransactions = async (fromAddress: Address, toAddressHash: AddressHash) => {
+export const buildSweepTransactions = async (fromAddressHash: AddressHash, toAddressHash: AddressHash) => {
   const { unsignedTxs } = await throttledClient.node.transactions.postTransactionsSweepAddressBuild({
-    fromPublicKey: fromAddress.publicKey,
+    fromPublicKey: keyring.exportPublicKeyOfAddress(fromAddressHash),
```

Comment:
In my goal to unify the `Address` types between the desktop wallet and the mobile wallet I realized that the only property that these 2 apps will eventually have different (after the mw migration to tanstack) is the `publicKey`. Desktop wallet includes it whereas mobile wallet doesn't. The reason mobile wallet doesn't is that it stores it in secure storage to avoid regenerating them on wallet unlock which is expensive for the mobile env. By relying on the keyring for public key access in the desktop wallet instead of the redux state we remove the duplicated state (until now the public key existed in both the keyring and the redux state) and also prepare for the unification of the 2 Address types between the wallets.